sass/layout/_post.scss

```diff
@@ -2110,3 +2110,82 @@
         pointer-events: none;
     }
 }
+
+.toast {
```

Comment:
Maybe it would make sense to put this next to the Toast component like we've started doing with the widgets? 0/5